### PR TITLE
Support different heading types

### PR DESCRIPTION
**Pull request by nlaz**
Update `Header` component with different heading types.

### CODE REVIEW

**Comment by Rovack** [comment]
Where are the styles for h1-h6 specified? Or are we just using the bootstrap headers?
If so, do they match the UX designs?

**Comment by nlaz** [comment]
@Rovack The default h1-h6 styles are specified by bootstrap. I believe they are close to the designs. Let me if there are changes you need.

**Comment by Rovack** [comment]
Ah alright, if we think they're close enough, then I don't have any other comments on this. Looks good.

**Comment by Rovack** [comment]
Oh actually @nlaz, one more thing I remembered: There are some screens that could use this new version of Header, which previously had to use raw divs with custom styles.
I know Location Name is like that, but there might also be others. Probably would be good to update them now to use this component.

**Comment by nlaz** [comment]
@Rovack @noidsign I updated the `h1`, `h2`, `h3`, and `p` global styles. I believe the header style on the location flow doesn't match any styles on the UI component list:
screen shot 2018-04-24 at 7 04 48 pm
screen shot 2018-04-24 at 7 05 32 pm

We should probably treat those headers differently in the meantime and push these changes forward. How does that sound?

**Comment by noidsign** [comment]
@nlaz 
+ @Rovack  FYI
The component for the title highlighted in the location flow screenshot is ```h4```  from the UI Components page.

File: src/index.css [hunk 1]
```diff
@@ -1,5 +1,40 @@
+@value darkerGray from "./colors.css";
+
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+h1 {
```
Comment (Rovack) [inline src/index.css]: Not that it matters too much, but do we not want these 3 in the CSS file for the Header component?

Comment (nlaz) [reply src/index.css]: They shouldn't be in the Header component because if someone uses `h1` without the component it should still have the same style.

Comment (Rovack) [reply src/index.css]: That's actually exactly what I was wondering about.
I would imagine generally no-one should be using raw tags, but if they do use e.g. a raw h1 header, bypassing our component, that would mean they don't want the standardization from the style guide. Just like if someone uses a raw input tag, bypassing our Input, they wouldn't get any of our styles for that.
But, I'm not too sure either way.

**Pull request merged**